test/go.mod

```diff
@@ -0,0 +1,130 @@
+module github.com/rancher-sandbox/rancher-turtles/test
+
+go 1.20
+
+replace github.com/rancher-sandbox/rancher-turtles => ../
+
+replace sigs.k8s.io/cluster-api-operator/test => github.com/Danil-Grigorev/cluster-api-operator/test v0.0.0-20230814131416-1045122a87bf
```

Comment:
Why are we doing the replace to your fork of CAPI operator?

Reply:
Good catch, perhaps before changes landing in capi operator this was needed

Reply:
Fixed